packages/platform/atoms/hooks/useHandleBookEvent.ts

```diff
@@ -75,6 +78,13 @@ export const useHandleBookEvent = ({
         ? duration
         : event.data.length;
 
+      if (values.responses) {
+        const prevResponse = Object.fromEntries(
+          Object.entries(values.responses).filter(([key]) => prevResponsesToStore.includes(key))
+        );
+        localStorage.setItem(`prevEventTypeResp`, JSON.stringify(prevResponse));
+      }
```

Comment:
It would be best to abstract getting and setting prevBookingResponse through a different module, so that it can be unit tested.